docs/Security/Protected Services/index.md

```diff
@@ -0,0 +1,36 @@
+# Protected Services
+
+#### Related Evolution Proposals
+- [0317: SDL Protocol Security Specification](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0317-sdl-protocol-security-specification.md)
+
+## 1. Overview
+
+Secured services in SDL are established using a TLS handshake, the TLS handshake process is defined by TLS and is not part of the SDL protocol.
+
+|||
+TLS Handshake Process
+![TLS Handshake activity diagram](https://user-images.githubusercontent.com/5848997/122258220-cb8b7100-ce9e-11eb-9b2a-a6194b0d1b68.png)

```

Comment:
Hmmm, seems this [isn't rendering properly](https://smartdevicelink.com/en/guides/pull_request/sdl-overview-guides/250cf04e5de7bbf6c19b72d99a91c3a7c0892463/security/protected-services/). Maybe you will need to include the image directly in the `assets` folder?

```suggestion
![TLS Handshake activity diagram](./assets/tls_handshake_process.png)
```

Reply:
@jacobkeeler added the assets folder and the link has been updated: https://github.com/smartdevicelink/sdl_overview_guides/pull/80/commits/c0bef995d1831312abdb33a52c6dff242a3d901b